Tests/Tests/CDTURLSessionTests.m

```diff
@@ -0,0 +1,62 @@
+//
+//  CDTURLSessionTests.m
+//  Tests
+//
+//  Created by Rhys Short on 24/08/2015.
+//
+//
+
+#import <XCTest/XCTest.h>
+#import "CloudantSyncTests.h"
+#import "CDTURLSession.h"
+#import "CDTHTTPInterceptorContext.h"
+#import "CDTHTTPInterceptor.h"
+
+@interface CDTNoOpRequestInterceptor : NSObject<CDTHTTPInterceptor>
+
+@property int timesCalled;
+
+@end
+@implementation CDTNoOpRequestInterceptor
```

Comment:
New line between @end and @implementation.

Excessive new lines within the implementation too.